browser/brave_rewards/android/brave_rewards_native_worker.cc

```diff
@@ -218,10 +218,8 @@ void BraveRewardsNativeWorker::OnGetRewardsParameters(
   }
 
   if (rewards_service) {
-    rewards_service->FetchBalance(
-      base::Bind(
-        &BraveRewardsNativeWorker::OnBalance,
-        weak_factory_.GetWeakPtr()));
+    rewards_service->FetchBalance(base::BindRepeating(
```

Comment:
@mariospr can you check this file also? I found that all methods from `RewardsService` receives once callback type?

Reply:
Oops! Another instance of this, sorry. Fixed now too (+ reviewed again the PR and couldn't find more problems)